src/core/test/dom.test.ts

```diff
@@ -27,6 +32,17 @@ describe('DOM test utils', () => {
         <button>2</button>
         <button>2</button>
       </div>
+      <ul class="${LIST_WITH_ITEMS_CLASS_NAME}">
+        <li class="${LIST_ITEM_CLASS_NAME} first-type">1</li>
+        <li class="${LIST_ITEM_CLASS_NAME} second-type">2</li>
+        <ul>
```

Comment:
Nitpick: nest this `ul` in a `li`.

Only `li`, `script` and `template` are permitted directly under `ul`: [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#technical_summary)